src/Command/SendAlertsCommand.php

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types=1);
+/**
+ * @package Divante\ProductAlert
+ * @author Marek Mularczyk <mmularczyk@divante.pl>
+ * @copyright 2020 Divante Sp. z o.o.
+ * @license See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Divante\ProductAlert\Command;
+
+use Divante\ProductAlert\Service\ProductAlertService;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class SendAlertsCommand
+ */
+class SendAlertsCommand extends Command
+{
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'product:alert:send';
+
+    /**
+     * @var ProductAlertService
+     */
+    private $productAlertService;
+
+    /**
+     * SendAlertsCommand constructor.
+     *
+     * @param ProductAlertService $productAlertService
+     * @param null $name
+     */
+    public function __construct(ProductAlertService $productAlertService)
```

Comment:
Ja bym serwisy wstrzykiwał jako Interfejsy. Będzie się to lepiej zgrywać jeżeli ktoś będzie chciał dekorować to później